README.md

```diff
@@ -235,7 +291,7 @@ This project received funding from the French ”Investing for the Future – PI
 
 ## 👨‍🎓 Creator
 
-This library was started as a side-project by [Thomas FEL](https://twitter.com/Napoolar) who is doing a thesis on explainability.
+This library was started as a side-project by [Thomas FEL](https://twitter.com/Napoolar) who is doing a thesis on explainability. He then received help from some members of the <a href="https://www.deel.ai/"> DEEL </a> team to enhance the library.
```

Comment:
Don't hesitate to put your name on it ;)

Reply:
Well let's go!